components/x-topic-search/src/AllFollowed.jsx

```diff
@@ -0,0 +1,29 @@
+import { h } from '@financial-times/x-engine';
+import styles from './TopicSearch.scss';
+import classNames from 'classnames';
+
+// transform like this => topic1, topic2 and topic3
+const transformFollowedTopics = followedTopicsIncludeSearchTerm => {
+	const topicsLength = followedTopicsIncludeSearchTerm.length;
+
+	return followedTopicsIncludeSearchTerm
+		.map((topic, index) => {
+			if (index + 1 === topicsLength) {
+				// the last topic
+				return  <span key={ index }>and <b>{ topic.name }</b></span>
+			} else {
+				if ((topicsLength - 2) === index) {
+					// one before the last topic
+					return <span key={ index }><b>{ topic.name }</b> </span>;
+				} else {
+					return <span key={ index }><b>{ topic.name }</b>, </span>;
+				}
+			}
+		})
+};
+
+export default ({ followedTopicsIncludeSearchTerm }) => (
```

Comment:
Would something like `matchingFollowedTopics` be better than `followedTopicsIncludeSearchTerm`? I'm trying to think of something a bit shorter...